query-parser/src/parser_util.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! matches_plain_symbol {
 /// The provided `$body` will be evaluated with `$input` bound to the input stream.
 ///
 /// `$body`, when run, should return a `ParseResult` of the appropriate result type.
+#[macro_export]
```

Comment:
I thought not putting these in the parser_utils crate in the first place was intentional.